ggml/src/ggml-cuda/CMakeLists.txt

```diff
@@ -98,6 +98,15 @@ if (CUDAToolkit_FOUND)
 
     set(CUDA_FLAGS -use_fast_math)
 
+    if (CUDAToolkit_VERSION VERSION_GREATER_EQUAL "12.8")
+        # Options are:
+        # - none (not recommended)
+        # - speed (nvcc's default)
+        # - balance
+        # - size
+        list(APPEND CUDA_FLAGS -compress-mode=${GGML_CUDA_COMPRESSION_MODE})
```

Comment:
Should this be `--compress-mode` instead? #12325

Reply:
according to the cuda docu both are accepted, and i chose single dash, because the other options next to it are also single dash.